bin/tape

```diff
@@ -25,6 +26,17 @@ opts.require.forEach(function(module) {
     }
 });
 
+if (opts.number !== false) {
+    var tape;
+    try {
+        tape = require(resolveModule("tape", { basedir: cwd }));
+    }
+    catch(err) {
+        tape = require("../"); // assume testing tape module itself
```

Comment:
I don't know. That's trick NPM stuff. That might allow you to replace the mysterious `var tape = require('../');` line found in each test file.